examples/chip-tool-webos/commands/clusters/ModelCommand.h

```diff
@@ -0,0 +1,61 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "../../config/PersistentStorage.h"
+#include "../common/CHIPCommand.h"
+#include <app/chip-zcl-zpro-codec.h>
+#include <lib/core/CHIPEncoding.h>
+
+class ModelCommand : public CHIPCommand
```

Comment:
This looks like a copy/paste of examples/chip-tool, without changes to most of the code, right?  If the problem is that chip-tool in its current form does not compile/run on webos (why not?), then the right fix is to fix that, not copy all the code.  Especially because this copy/paste is already out of date; chip-tool has changed under it.

Reply:
@bzbarsky-apple 
First of all, I appreciate your kind review.

webos has no problem compiling the examples/chip-tool.
We just intended to add webos specific changes on chip-tool. That's why I created separate example folder for webos.

But I think your review is right.
In this case, chip-tool resources in the examples/chip-tool-webos will be out-of-date which will introduce build or runtime error in the future.

Referencing other platform, it seems like they set their build to use the latest chip-tool located on examples/chip-tool.
I think webos should follows the same.

Let me check your review and will get back to you once fix is ready.
Thank you.

Reply:
To keep the chip-tool compiled on webOS as up-to-date, I decided to remove the examples/chip-tool-webos.
From now on, examples/chip-tool will be built as the example for webOS OSE.
Thank you.